SYCL/SubGroup/scan_spirv13_fp16.cpp

```diff
@@ -0,0 +1,20 @@
+// UNSUPPORTED: cuda
+
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+// This test verifies the correct work of SPIR-V 1.3 exclusive_scan() and
+// inclusive_scan() algoriths used with the MUL operation.
+
+#include "scan.hpp"
+
+int main() {
+  queue Queue;
+  if (!core_sg_supported(Queue.get_device())) {
```

Comment:
shouldn't we check that "cl_khr_fp16" extension is supported?

Reply:
Fixed in 925ce426bb754423531f70b26482e9c98005857a Thank you!